src/libPMacc/include/Environment.hpp

```diff
@@ -240,12 +240,34 @@ class Environment
 
 /* No namespace for macro defines */
 
+/** start a dependency chain */
 #define __startTransaction(...) (PMacc::Environment<>::get().TransactionManager().startTransaction(__VA_ARGS__))
-#define __startAtomicTransaction(...) (PMacc::Environment<>::get().TransactionManager().startAtomicTransaction(__VA_ARGS__))
+
+/** end a opened dependency chain */
 #define __endTransaction() (PMacc::Environment<>::get().TransactionManager().endTransaction())
+
+/** mark the begin of an operation
+ *
+ * depended on the opType this method is blocking
+ *
+ * @param opType place were the operation is running
+ *               possible places are: `ITask::TASK_CUDA`, `ITask::TASK_MPI`, `ITask::TASK_HOST`
+ */
 #define __startOperation(opType) (PMacc::Environment<>::get().TransactionManager().startOperation(opType))
+
+/** get a `EventStream` that must be used for cuda calls
+ *
+ * depended on the opType this method is blocking
+ *
+ * @param opType place were the operation is running
```

Comment:
"place" seems to be a bad choice. It suggests that CUDA means on the device or so. "Subsystem" might be better.